app/visit/components/checkIn.js

```diff
@@ -6,7 +6,7 @@
     .component('checkIn', {
       bindings: {
         patient: '<',
-        onCheckIn: '&'
+        onCheckInChange: '&'
```

Comment:
Just curious: does this bind ```vm.onCheckInChange``` to the ```on-check-in-change``` attribute, so when you call ```vm.onCheckInChange``` below it runs the attribute's value (in this case ```vm.reload()```)?

Reply:
Correct. It's a way of having communication between components, you can pass values from child components to its parents using this kind of binding.

https://docs.angularjs.org/guide/directive#creating-a-directive-that-wraps-other-elements